lib/reducers/auth/store_user.dart

```diff
@@ -5,14 +5,13 @@ import 'package:crowdleague/models/navigation/page_data/initial_page_data.dart';
 import 'package:crowdleague/models/navigation/page_data/page_data.dart';
 import 'package:redux/redux.dart';
 
-/// Set the user object to null if action has null user or update the state
```

Comment:
user object should never be null (if it is, then clearUserData action would be dispatched) so removed nullcheck + updated comment

Reply:
Beautiful!